.github/workflows/installer-macos.yml

```
@@ -38,21 +38,12 @@ jobs:
           if [[ -z ${LITE_FLAG} ]]; then
               INSTALL_FLAGS+=('-r' 'req-scientific.txt')
           fi
-          ${pythonLocation}/bin/python -m pip install -U pip setuptools wheel
+          ${pythonLocation}/bin/python -m pip install -U pip setuptools
           ${pythonLocation}/bin/python -m pip install -r req-build.txt -r req-extras.txt -r req-plugins.txt "${INSTALL_FLAGS[@]}" -e ${GITHUB_WORKSPACE}
           ${pythonLocation}/bin/python -m pip uninstall -q -y spyder
       - name: Install Subrepos
         if: ${{github.event_name == 'pull_request'}}
-        run: |
-          for dep in $(ls ${GITHUB_WORKSPACE}/external-deps)
-          do
-              if [ "$dep" = "python-lsp-server" ]; then
-                  SETUPTOOLS_SCM_PRETEND_VERSION=`${pythonLocation}/bin/python ${GITHUB_WORKSPACE}/pylsp_utils.py` \
-                  ${pythonLocation}/bin/python -m pip install ${GITHUB_WORKSPACE}/external-deps/$dep
-              else
-                  ${pythonLocation}/bin/python -m pip install ${GITHUB_WORKSPACE}/external-deps/$dep
-              fi
-          done
+        run: ${pythonLocation}/bin/python -bb -X dev -W error ${GITHUB_WORKSPACE}/install_subrepos.py --overwrite-standard
```

--overwrite-standard is not necessary.
```suggestion
        run: ${pythonLocation}/bin/python -bb -X dev -W error ${GITHUB_WORKSPACE}/install_subrepos.py
```